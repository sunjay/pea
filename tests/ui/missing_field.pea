struct Cow {
    name: [u8],
}

fn main() {
    let cow = Cow {};
}
