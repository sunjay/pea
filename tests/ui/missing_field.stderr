[tests/ui/missing_field.pea:6:15-6:17] error: missing field `name` in initializer of `Cow`

error: aborting due to 1 previous error

